src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs

```diff
@@ -1,5 +1,6 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Linq;
```

Comment:
classification is super perf sensitive and is called a bunch.  We've avoided linq here because of that.